File: LegacyToQtd2Migrator/QTD2ScormContext/Employee.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace LegacyToQtd2Migrator.QTD2ScormContext
{
    public partial class Employee
    {
        public Employee()
        {
            ClassScheduleEmployees = new HashSet<ClassScheduleEmployee>();
        }

        public int Id { get; set; }
        public int PersonId { get; set; }
        public string EmployeeNumber { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string ZipCode { get; set; }
        public string PhoneNumber { get; set; }
        public string WorkLocation { get; set; }
        public string Notes { get; set; }
        public string Password { get; set; }
        public bool Tqequlator { get; set; }
        public bool? Deleted { get; set; }
        public bool Active { get; set; }
        public string CreatedBy { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string ModifiedBy { get; set; }
        public DateTime? ModifiedDate { get; set; }
        public DateTime? InactiveDate { get; set; }
        public string Reason { get; set; }

        public virtual Person Person { get; set; }
        public virtual ICollection<ClassScheduleEmployee> ClassScheduleEmployees { get; set; }
    }
}

File: QTD.SPA/src/app/_DtoModels/Employee/Employee.ts
import { EmployeeCertification } from '../EmployeeCertification/EmployeeCertification';
import { EmployeeOrganization } from '../EmployeeOrganization/EmployeeOrganization';
import { EmployeePosition } from '../EmployeePosition/EmployeePosition';
import { Entity } from '../Entity';
import { Person } from '../Person/Person';

export class Employee extends Entity {
  personId!: any;
  employeeNumber!: string;
  address: string;

  city: string;

  state: string;

  zipCode: string;

  phoneNumber: number;

  workLocation: string;

  notes: string;

  password: string;

  tqEqulator: boolean;

  idpReviewInformation?:string;

  isCertificationRequired!:boolean;
  person!: Person;
  employeePositions!: EmployeePosition[];
  employeeOrganizations!: EmployeeOrganization[];
  employeeCertifications!: EmployeeCertification[];
  classSchedule_Employee:number;
  taskQualifications:any[];
  inactiveDate:any;
  reason:any;
  publicUser: boolean;
}
